projects/core/src/occ/occ-models/occ.models.ts

```diff
@@ -4276,4 +4276,31 @@ export namespace Occ {
     pagination?: PaginationModel;
     sorts?: SortModel[];
   }
+
+  export interface OrgUnitCustomer {
+    active?: boolean;
+    approvers?: any[];
+    currency?: Currency;
+    customerId?: string;
+    deactivationDate?: string;
+    defaultAddress?: Address;
+    displayUid?: string;
+    email?: string;
+    firstName?: string;
+    language?: Language;
+    lastName?: string;
+    name?: string;
+    orgUnit?: B2BUnit;
+    roles?: string[];
+    selected?: boolean;
+    title?: string;
+    titleCode?: string;
+    uid?: string;
+  }
+
+  export interface OrgUnitCustomerList {
+    orgUnitCustomers: OrgUnitCustomer[];
+    pagination?: PaginationModel;
+    sorts?: SortModel[];
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/8040837/76858550-f203cc00-6857-11ea-8836-80113c8c27f3.png)

```suggestion
  export interface OrgUnitUserList {
    orgUnitCustomers: B2BUser[];
    pagination?: PaginationModel;
    sorts?: SortModel[];
  }
```